docs/internals/layout_in_memory.rst

```diff
@@ -36,4 +36,41 @@ elements.
   definitely zeroed out memory area, using such a pointer non-temporarily
   without updating the free memory pointer can have unexpected results.
 
-.. index: calldata layout
\ No newline at end of file
+
+Differences to Layout in Storage
+================================
+
+As described above the layout in memory is different from the layout in
+:ref:`storage<storage-inplace-encoding>`. Below there are some examples.
+
+Example for Difference in Arrays
+--------------------------------
+
+The following array occupies 32 bytes (1 slot) in storage, but 128
+bytes (4 items with 32 bytes each) in memory.
+
+::
+
+    uint8[4] a;
+
+
+
+Example for Difference in Struct Layout
+---------------------------------------
+
+The following struct occupies 96 bytes (3 slots of 32 bytes) in storage,
+but 128 bytes (4 items with 32 bytes each) in memory.
+
+
+::
+
+    struct S{
+        uint a;
+        uint b;
+        uint8 c;
+        uint8 d;
+    }
+
+
+
+.. index: calldata layout
```

Comment:
It seems this line belong to the top of `layout_in_calldata` - can you move it there, please?

Reply:
Thanks @chriseth ! Fixed in 8a2731bfdb32aa81a84d6da3ef211958073bae4a